keycloak.yaml

```diff
@@ -76,8 +76,18 @@ advisories:
     - timestamp: 2023-04-09T09:54:02.782617-04:00
       status: not_affected
       justification: vulnerable_code_not_in_execute_path
+  CVE-2023-0264:
+    - timestamp: 2023-04-30T07:15:20.601606-04:00
+      status: not_affected
+      justification: component_not_present
+      impact: The CVE was fixed in a version of keycloak prior to our packaging
+      fixed-version: "0"
+    - timestamp: 2023-04-30T07:15:51.679089-04:00
+      status: not_affected
+      justification: vulnerable_code_not_in_execute_path
```

Comment:
Is having both justification's here intentional?

Reply:
It shows up during sync secfixes, I was confused too but it happens above for the others as well.

Reply:
Okay, got it. That could be a bug then... 🤔  I'll take a look soon.

My suggestion here would be to have this for all three advisories:

```yaml
status: not_affected
justification: vulnerable_code_not_present
impact: The CVE was fixed in a version of keycloak prior to our packaging
```

When I delete the `secfixes:` section, and update the advisories to just this ^, sync-secfixes gives me something that looks right. **I'll push a commit** that's just a suggestion, but **feel free to drop** the commit if you disagree.

Reply:
Ah nice!